profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -327,3 +441,60 @@ function _tmgmt_workbench_clear_request_languages() {
     unset($_SESSION['tmgmt_workbench']['languages']);
   }
 }
+
+/**
+ * Injects 'request additional languages' functionality form elements.
+ *
+ * @param array $form
+ *   Reference to the main form array.
+ * @param array $trans_langs
+ *   An array with translatable languages.
+ */
+function _tmgmt_workbench_inject_add_new_languages_elements(&$form, $trans_langs) {
+  // Setting up header for the additional languages.
+  $trans_lang_header = array(
+    'language' => t('Language'),
+  );
+
+  // Setting up $options for the additional languages.
+  foreach ($trans_langs as $lg_code => $trans_lang) {
+    $trans_langs_options[$lg_code]['language'] = $trans_lang;
+  }
+
+  $form['add_new_languages'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Request addition of new languages'),
+    '#prefix' => '<div class="tmgmt-poetry-ui-add-new-languages">',
+    '#suffix' => '</div>',
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#tree' => FALSE,
+  ];
+
+  $form['add_new_languages']['translatable_languages'] = [
+    '#title' => t('Additional languages'),
+    '#type' => 'tableselect',
+    '#header' => $trans_lang_header,
+    '#options' => $trans_langs_options,
+    '#empty' => t('No content available.'),
+    '#prefix' => '<div class="col-lg-4, col-md-4">',
+    '#suffix' => '</div>',
+  ];
+
+  $form['add_new_languages']['delai'] = [
+    '#type' => 'date_popup',
```

Comment:
If you use date_popup form api element pls make the module depends on date_popup (even it's enabled platform-wide, but still semantically it's important)